proposals/csharp-7.2/readonly-ref.md

```diff
@@ -566,7 +566,7 @@ static readonly ref Vector3 M1_Trace()
 Except for their `readonly` nature, `ref readonly` locals behave like ordinary `ref` locals and are subject to exactly same restrictions.  
 For example restrictions related to capturing in closures, declaring in `async` methods or the `safe-to-return` analysis equally applies to `ref readonly` locals.
 
-# Ternary `ref` expressons. (aka "Conditional LValues")
+# Ternary `ref` expressions. (aka "Conditional LValues")
```

Comment:
(Not sure if it is wise to fix this typo, as it might break existing hyperlinks.)